hack/boskos.py

```diff
@@ -37,7 +41,15 @@ def checkout_account(resource_type, user, input_state="free"):
         print(f"export BOSKOS_RESOURCE_NAME={result['name']}")
         print(f"export AWS_ACCESS_KEY_ID={result['userdata']['access-key-id']}")
         print(f"export AWS_SECRET_ACCESS_KEY={result['userdata']['secret-access-key']}")
-
+    # The http API has two possible meanings of 404s - the named resource type doesn't exist or there no available resources of the type.
+    # For our purposes, we don't need to differentiate.
+    elif r.status_code == 404:
+        print(f"could not find available host, retrying in {RETRY_WAIT}s")
```

Comment:
The 404 may also be because the resource is not found as well.

Reply:
I think based on what we saw earlier this week. But i will check for sure.

Reply:
It does also return a 404 if the resource is not found: https://github.com/kubernetes-sigs/boskos/blob/master/handlers/handlers.go#L78

Reply:
I'll clarify this comment, `named resource type doesn't exist` was supposed to cover this case